### PR TITLE
Stabilization and better rendering

### PR DESCRIPTION
**Pull request by NastuzziSamy**
- Overlay all the page to detect all clicks (to close the menu). It is necessary because some events do not propagate the click. 
- Remove regular click option as "Open folder", "Open in new tab" => needs to be removed from Transifex
- Hide blinking menus and make them usable in the right time.

### CODE REVIEW

File: js/script.js [hunk 1]
```diff
@@ -393,14 +388,11 @@ var RightClick = RightClick || {};
         return true;
     };
 
-    exports.isAMenuOpened = function (key) {
-        for (key; key < exports.menus.length; key++) {
+    exports.isAMenuOpened = function () {
+        for (let key = 0; key < exports.menus.length; key++) {
```
Comment (skjnldsv) [inline js/script.js]: is this code compiled afterwise? Because this is not compatible with ie11 :(

Comment (NastuzziSamy) [reply js/script.js]: Ah yes, it is not, but it should ^^

Thx ! :)

**Comment by MorrisJobke** [comment]
> * Remove regular click option as "Open folder", "Open in new tab" => needs to be removed from Transifex

This will be done automatically by the transifex sync job 😉

**Comment by NastuzziSamy** [comment]
If it is ok for you @skjnldsv @jancborchardt @MorrisJobke  @rullzer, I can merge this.

It will fix also that issue: https://github.com/nextcloud/files_rightclick/issues/56

**Comment by NastuzziSamy** [comment]
Hey Bart @skjnldsv :) It's Samy* ^^

It is handled now, dont know why I used an absolute position x)

**Pull request merged**